### PR TITLE
eliminate hardcoded tags and clean up module

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      run: |
+        brew install docker tfenv tflint
+        tfenv install
+
+    - name: tf fmt
+      run: |
+        terraform fmt
+    - name: tflint
+      run: |
+        tflint
```
Comment (sblack4) [inline workflows/check.yml:20]: do we still want to be running `tflint` if we're taking it out of the pre-commit hook?

**Review by sblack4 — APPROVED** [review]
👍

File: workflows/check.yml (hunk 1 above)
Comment (cdaniluk) [reply workflows/check.yml:20]: I figured yes, because it seems to run successfully in the GH action.

Comment (sblack4) [reply workflows/check.yml:20]: If it works it works

**Pull request merged**